package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.6.0",
+    "@emotion/styled": "^11.6.0",
+    "@mui/material": "^5.2.1",
```

Comment:
this libraries where approved by your teammates?

Reply:
We agreed on working with MUI. I don't know if @emotion/react and @emotion/styled are required to work with MUI. Just followed the instructions from the documentation.